############################################################################ Help ###################################################################################

Vinifera is a graphical single-cell ATAC-seq (scATAC-seq) analysis pipeline for genomics scientists. The pipeline will graphically guide you through the analysis of scATAC-seq data, starting from peakfile, bamfile and Metafile . It uses a comprehensive set of modules for quality control, clustering, differentially accessible regions,motif discovery, co-accessibility, gene Accessibility Score, cell pseudo-time pathway construction and enrichment analysis.

Home
You should provide peakfile,bamfile and Metafile as input.

Peakfile is the bed format and contains at least three columns. The first column is the chromosome number, the second column is the starting site, and the third column is the termination site
chr1	713876	714376
chr1	762653	763153

BAM file is the binary format of Sam format. SAM file is the file generated by short sequence alignment. It is the core file in the second generation sequencing
Metafile is the unique classification, source and other characteristic information of each data set

Quality-Control
Personalize the threshold to filter out samples with insufficient readings or low proportion of peak readings, as they may represent empty holes or dead cells. Filtering operation can improve the accuracy of subsequent analysis.

filterSamples {chromVAR}
min_in_peaks	:minimum fraction of samples within peaks
min_depth	:minimum library size

filterPeaks {chromVAR}
min_fragments_per_peak	:minimum number of fragmints in peaks across all samples

Motif-Accessibility-Deviation
The first application is simply to compute the variability of each motif or annotation across the cells or samples of interest.

getJasparMotifs {chromVAR}
Collection:
  "CORE":Under this category, the eukaryotic transcription factor motif information is collected from the literature and supported by experimental evidence. After manual verification, it is a non redundant and high-quality transcription factor motif database, so it is also the core of the whole database. Because of its high quality, non redundancy and other characteristics, this category information is usually our first choice.
"CNE":The data set contains 233 transcription factor motif information regulating human non coding genes, which is collected and sorted according to the data in Xie et al. (PNAs 2007), and the number starts with CN
"PHYLOFACTS":The transcription factor motif information of mammalian evolutionarily conserved genes is analyzed under this category
"SPLICE":This category contains motif sequences of human shear sites, with a small amount of data. There are only 6 motifs in total, and each motif number starts with SA
"POLII":This category contains motif sequences of RNA polymerase binding regions, and each motfi number starts with pl
"FAM":The category information of transcription factors is stored under this category. Multiple transcription factors can have the same regulatory sequence, and the transcription factors with the same regulatory sequence are classified into one category. The number of each class starts with MF
"PBM":Under this category is the motif information obtained after analyzing the transcription factors of 104 mice by in vitro technology. Each motif number starts with Pb.

Sample-Similarity
We can also use the bias corrected deviations to cluster the samples. The function getSampleCorrelation first removes highly correlated annotations and low variability annotations and then computes the correlation between the cells for the remaining annotations.

cutree {stats}
expect_cell_number:an integer scalar or vector with the desired number of groups

Clustering
The dimensionality reduction clustering of single cells using the data of scatac SEQ may find new subsets that can not be found by conventional RNA sequencing clustering; It can also be cross verified with RNA sequencing clustering to improve the accuracy of single cell clustering.

We provide two clustering methods: 1. Consider the number of reads on each peak. 2. Only consider whether chromatin is open. The results obtained by the two methods can be used for cross validation and selective analysis.

Variability
Here is the difference accessible area between the two groups selected by the user. The user can set the maximum acceptable adjusted p-value for data filtering. This table not only shows the starting site and chromosome of the difference accessible region, but also includes the difference multiple and adjusted p-value data.

Motif-Similarity
A large number of motifs have been obtained from previous studies. Due to the large number, it is not convenient for researchers to select. Here, the similarity analysis of the main motifs is carried out, and they are clustered and presented to facilitate users to select the motif set of interest.

deviationsTsne {chromVAR}
Threshold:variability threshold â€“ use only deviations with variability greater than threshold
Perplexity:perplexity parameter for tsne

Co-accessibility
Calculate the co accessibility between paired genes. Evaluate the co accessibility between genes according to the score. The higher the score, the greater the co accessibility of genes, indicating that the possibility of co expression of these two genes is greater. Draw a visual chart according to the evaluation results.

At the same time, we also calculated the gene activity score. The higher the score, the greater the gene accessibility, indicating the higher the gene expression activity in a certain period of time.

Accessibility-Trajectories
During development, cells respond to stimuli, and cells in different states express different genes, so as to complete their work. These transient states are often difficult to characterize because it is difficult or impossible to purify cells between more stable states. Pseudo timing analysis of single-cell ATAC SEQ can view these states without purifying cells.

Pathway
Using GO database and rGREAT package, we can get the enrichment of our target genes at three levels: cellular component (CC), molecular function (MF) and biological process (BP).